lib/internal/bootstrap_node.js

```diff
@@ -312,62 +297,48 @@
 
   function setupGlobalConsole() {
     const originalConsole = global.console;
-    let console;
+    // Setup inspector command line API
+    const { addCommandLineAPI, consoleCall } = process.binding('inspector');
```

Comment:
needs `if (!addCommandLineAPI) return;`?

Reply:
Oh, you are right, this part should be moved below the `if (!consoleCall)` statement. That should be sufficient.